Src/FluentAssertions/Specialized/ActionAssertions.cs

```diff
@@ -114,6 +114,60 @@ public void NotThrow(string because = "", params object[] becauseArgs)
             }
         }
 
+        /// <summary>
+        /// Asserts that the current <see cref="Action"/> stops throwing any exception
+        /// after a specified amount of time.
+        /// </summary>
+        /// <remarks>
+        /// The <see cref="Action"/> is invoked. If it raises an exception,
+        /// the invocation is repeated until it either stops raising any exceptions
+        /// or the specified wait time is exceeded.
+        /// </remarks>
+        /// <param name="waitTime">
+        /// The time after which the <see cref="Action"/> should have stopped throwing any exception.
+        /// </param>
+        /// <param name="pollInterval">
+        /// The time between subsequent invocations of the <see cref="Action"/>.
+        /// </param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        /// <exception cref="ArgumentOutOfRangeException">Throws if waitTime or pollInterval are negative.</exception>
+        public void NotThrowAfter(TimeSpan waitTime, TimeSpan pollInterval, string because = "", params object[] becauseArgs)
+        {
+            FailIfSubjectIsAsyncVoid();
+
+            if(waitTime < TimeSpan.Zero)
+                throw new ArgumentOutOfRangeException(nameof(waitTime), $"The value of {nameof(waitTime)} must be positive.");
+
+            if(pollInterval < TimeSpan.Zero)
+                throw new ArgumentOutOfRangeException(nameof(pollInterval), $"The value of {nameof(pollInterval)} must be positive.");
+
+            TimeSpan? invocationEndTime = null;
+            Exception exception = null;
+
+            var watch = Stopwatch.StartNew();
+
+            using(var waiter = new ManualResetEvent(false)) {
```

Comment:
🙃 brace should be on new line.
If your editor supports [EditorConfig](https://editorconfig.org/), formatting the entire document should automatically use the project preferences. 
I've recently updated the `.editorconfig` file which defines this, so rebase on master to get the best auto format experience.

Reply:
I will check it out.